.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     environment:
       GOCACHE: "/tmp/go/cache"
-      GOLANGCI_LINT_VERSION: "1.43.0"
+      GOLANGCI_LINT_VERSION: "1.46.2"
```

Comment:
We have moved from CircleCI to GitHub actions. These upgrades are now handled by dependabot (example PR https://github.com/helm/helm/pull/30535)